### PR TITLE
feat: added Artemis bundles and resources

### PR DESCRIPTION
**Pull request by marcellorinaldo**
This PR contributes the Apache ActiveMQ Artemis MQTT broker from the Eclipse Kura repo (https://github.com/eclipse-kura/kura).

Documentation: https://github.com/eclipse-kura/kura-artemis/pull/3

### CODE REVIEW

File: bundles/org.apache.activemq.artemis/pom.xml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>6.0.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Include-Resource>
+                            ${project.basedir}/lib
```
Comment (nicolatimeus) [inline bundles/org.apache.activemq.artemis/pom.xml]: Why not using `Embed-Dependency` as done in the Kura repo?

Comment (marcellorinaldo) [reply bundles/org.apache.activemq.artemis/pom.xml]: Just to follow the same convention that we used on other projects of the kura target platform, if you feel it is better, we can use the embed-dependency

Comment (nicolatimeus) [reply bundles/org.apache.activemq.artemis/pom.xml]: I think you can use embed dependency, it is specific for this purpose and more compact. The other convention is probably the only one possible with Tycho

**Comment by marcellorinaldo** [comment]
Sonar is failing for missing tests (there were none also in kura-core main repo).

File: bundles/org.eclipse.kura.broker.artemis.core/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/bin/
```
Comment (salvatore-coppola) [inline bundles/org.eclipse.kura.broker.artemis.core/.gitignore]: maybe we can use a single .gitignore in the root e move these rules in it

File: bundles/org.eclipse.kura.broker.artemis.core/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=17))"
+Import-Package: org.apache.activemq.artemis.core.config;version="2.6.0",
+ org.apache.activemq.artemis.core.config.impl;version="2.6.0",
+ org.apache.activemq.artemis.core.deployers;version="2.6.0",
+ org.apache.activemq.artemis.core.server;version="2.6.0",
+ org.apache.activemq.artemis.jms.server.config.impl;version="2.6.0",
+ org.apache.activemq.artemis.spi.core.protocol;version="2.6.0",
+ org.apache.activemq.artemis.spi.core.security;version="2.6.0",
+ org.apache.activemq.artemis.spi.core.security.jaas;version="2.6.0",
+ org.eclipse.kura.configuration;version="[1.1,2)",
```
Comment (salvatore-coppola) [inline bundles/org.eclipse.kura.broker.artemis.core/META-INF/MANIFEST.MF]: miss a .0 here

File: bundles/org.eclipse.kura.broker.artemis.simple.mqtt/.gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+/bin/
```
Comment (salvatore-coppola) [inline bundles/org.eclipse.kura.broker.artemis.simple.mqtt/.gitignore]: same here for .gitignore

**Review by salvatore-coppola — DISMISSED** [review]

**Review by salvatore-coppola — APPROVED** [review]

**Comment by marcellorinaldo** [comment]
@MMaiero Need a force merge on this. Tests are missing since not present in the Kura codebase.

**Comment by MMaiero** [comment]
Forcing merge as per requested and matching the contribution of existing code
